README.md

```diff
@@ -0,0 +1 @@
+# VPC-Deployment-using-Terraform
```

Comment:
We don't yet need a readme file. Please remove it

Reply:
readme file removed